### DISCUSSION TITLE
audio recorder

### DISCUSSION THREAD

**Discussion by meichthys** [TriliumNext/Trilium#5962] [Plugin Ideas]
zadam/trilium#1389
zadam/trilium#1924

**Comment by eliandoran** [comment]
Interesting idea, but I guess it will be better off implemented as a plugin once we reach that milestone.

**Comment by diyoyo** [comment]
I think now we could even include ( in the plugin) AI-driven VoiceToText and VoiceToSummary 😉 at that point
